lib/src/components/NxTag/NxTag.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+import { faPlusCircle } from '@fortawesome/free-solid-svg-icons';
+
+import './NxTag.scss';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTag: FunctionComponent<Props> =
+    function NxTag(props) {
+      const { children, className, tagColor, ...attrs } = props,
+          tagClasses = classnames('nx-tag', className, {
+            [`nx-tag--${tagColor}`]: tagColor,
+            ['nx-tag--default']: !tagColor
+          });
+
+      return (
+        <div className={tagClasses} {...attrs}>
+          <NxOverflowTooltip>
```

Comment:
If you follow my advice in https://github.com/sonatype/sonatype-react-shared-components/pull/290/files#r562783284 , you'll want to move the `NxOverflowTooltip` to wrap the entire `NxTag`.

Reply:
> If you follow my advice in https://github.com/sonatype/sonatype-react-shared-components/pull/290/files#r562783284 , you'll want to move the `NxOverflowTooltip` to wrap the entire `NxTag`.

Done for both: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/93a4c999da97853d1188164e4731770c4052d47b